src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceTest.java

```diff
@@ -172,9 +260,9 @@ void rotateMasterKeyNewEnough(Vertx vertx, VertxTestContext testContext) throws
 
         final EncryptionKey[] keys = {
                 new EncryptionKey(11, null, Instant.ofEpochMilli(KEY_CREATE_TIME_IN_MILLI), Instant.ofEpochMilli(KEY_ACTIVATE_TIME_IN_MILLI), Instant.ofEpochMilli(KEY_EXPIRE_TIME_IN_MILLI), -1),
-                new EncryptionKey(12, null, Instant.ofEpochMilli(KEY_CREATE_TIME_IN_MILLI+1), clock.now().plusSeconds(MASTER_KEY_ACTIVATES_IN_SECONDS + 1000), Instant.MAX, -1),
+                new EncryptionKey(12, null, Instant.ofEpochMilli(KEY_CREATE_TIME_IN_MILLI+1), clock.now().plusSeconds(MASTER_KEY_ACTIVATES_IN_SECONDS + TEN_DAYS_IN_SECONDS * 10), clock.now().plusSeconds(MASTER_KEY_EXPIRES_AFTER_SECONDS + TEN_DAYS_IN_SECONDS * 10), -1),
```

Comment:
TEN_DAYS_IN_SECONDS is already defined as 864000L (86400 a day * 10 days), why does it * 10 again?

Reply:
or if you want 100 days might be better to define A_HUNDRED_DAYS_IN_SECONDS for readability?

Reply:
You are right! I will make that change.